Assets/Scripts/Dialogue scripts/Dialogue.cs

```diff
@@ -1,197 +1,184 @@
-public class Dialogue : Interacted
+namespace Dialogue_scripts
 {
-    [TextArea] public string[] _textBox; // string array for inputting separate dialogue boxes
-
-    [TextArea] public string[] _hiddenTextBox;
-
-    [SerializeField] private RenderDialogue _pageRender;
-
-    [SerializeField] GameObject rayCast;
-
-    [SerializeField] GameObject dialogueBox;
-
-    public JournalInventoryScript joInvScript;
-
-    [SerializeField] GameObject accessItem;
-
-    [SerializeField] GameObject hiddenItem;
-
-    [SerializeField] GameObject item;
-
-    [SerializeField] private DeathMovement _movement;
-
-    public int _itemClass;
-
-    public int _pageClass;
-
-    BoxCollider2D _myCollider;
-
-    SpriteRenderer _myRenderer;
-
-    public bool isIntro = false;
-
-
-    private void Start()
+    [RequireComponent(typeof(BoxCollider2D))]
+    public class Dialogue : Interacted
     {
-        dialogueBox.SetActive(false);
+        [TextArea] [SerializeField] private string[] _textBox; // string array for inputting separate dialogue boxes
+        [TextArea] [SerializeField] private string[] _hiddenTextBox;
+        [SerializeField] private RenderDialogue _pageRender;
+        [SerializeField] private GameObject _rayCast;
+        [SerializeField] private GameObject _dialogueBox;
+        [SerializeField] private JournalInventoryScript _joInvScript;
+        [SerializeField] private GameObject _accessItem;
+        [SerializeField] private GameObject _hiddenItem;
+        [SerializeField] private GameObject _item;
+        [SerializeField] private DeathMovement _movement;
+        [SerializeField] private int _itemClass;
+        [SerializeField] public int pageClass;
+        private BoxCollider2D _myCollider;
+        private SpriteRenderer _myRenderer;
+        [SerializeField] private bool isIntro;
+
+
+        private void Start()
+        {
+            _dialogueBox.SetActive(false);
 
-        _myCollider = GetComponent<BoxCollider2D>();
+            _myCollider = GetComponent<BoxCollider2D>();
 
-        _myRenderer = gameObject.GetComponent<SpriteRenderer>();
+            _myRenderer = gameObject.GetComponent<SpriteRenderer>();
 
-        if (hiddenItem != null) hiddenItem.SetActive(false);
+            if (_hiddenItem != null) _hiddenItem.SetActive(false);
 
-        if (item != null) item.SetActive(false);
-    }
+            if (_item != null) _item.SetActive(false);
+        }
 
 
 
-    public void Interact(DeathMovement playerInteraction)
-    {
-        if (_movement != null) _movement.enabled = false;
-        if (accessItem != null)
+        public void Interact(DeathMovement playerInteraction)
         {
-            if (accessItem.activeSelf == false) StartCoroutine(RunParagraphCycle());
+            if (_movement != null) _movement.enabled = false;
+            if (_accessItem != null)
+            {
+                if (_accessItem.activeSelf == false) StartCoroutine(RunParagraphCycle());
```

Comment:
I tend to prefer braces when an `if` has a matching `else` block, but that is purely preference.